sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>"the ciphtertext is what you should <s>kept</s> keep"</p><p> </p><p>"nothing preventing you from <s>encrypt</s> encrypting"</p><p> </p><p>"you used to <s>encrypted</s> encrypt it"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>